### PR TITLE
Feature/Update gitignore

### PR DESCRIPTION
**Pull request by JorgeGreek**
update gitignore for .idea files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,9 @@ out/
 .gradle/
 build/
 
+# Idea files
+.idea
```
Comment (juancoob) [inline .gitignore]: Hello @JorgeGreek all idea files are unnecessary?

Comment (JorgeGreek) [reply .gitignore]: They are not unnecessary but I understood that they do not need to be committed

**Review by juancoob — APPROVED** [review]
Thanks @JorgeGreek !!

**Review by joseantoniocampillo — APPROVED** [review]
It's good. I see only at this point remove files that come back and now it is in .gitignore, like .idea/compiler.xml

**Pull request merged**